dali/pipeline/data/tensor.h

```diff
@@ -40,6 +41,32 @@ class Tensor : public Buffer<Backend> {
   inline Tensor() : meta_(DALI_NHWC) {}
   inline ~Tensor() = default;
 
+  /**
+   *
+   * @brief For tensor T of shape (s_0, s_1, ..., s_{n-1}) returns a n-1 dimensional tensor T'
+   *        of shape (s_1, s_2, ..., s_{n-1}), such that
+   *        T'(x_1, x_2, ..., x_{n-1}) = T(x, x_1, x_2, ..., x_{n-1})
+   *        for param 'x' and any valid x_1, x_2, ..., x_{n-1}
+   *
+   * Returned tensor is treated as a view to this tensors and shares memory with it.
+   * @param x
+   * @return Tensor<Backend>
+   */
+  Tensor<Backend> SubspaceTensor(Index x) {
```

Comment:
I suppose, that this function should have some tests

Reply:
Done

Reply:
Do we want to keep this with the upcoming refactor?

Reply:
When we will have something better, we will replace this, I am leaving it for now.